src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react';
+import { InjectAppServices } from '../../../services/pure-di';
+import { FormattedMessage } from 'react-intl';
+import Loading from '../../Loading/Loading';
+import C3Chart from '../../shared/C3Chart/C3Chart';
+
+let chartConfig = {
+  data: {
+    json: null,
+    type: 'spline',
+    keys: {
+      x: 'from',
+      value: ['quantity'],
+    },
+  },
+  legend: {
+    show: false,
+  },
+  axis: {
+    x: {
+      type: 'timeseries',
+      tick: {
+        format: '%Y-%m-%d',
+      },
+    },
+  },
+};
+
+const ReportsDailyVisits = ({ domainName, dateFrom, dependencies: { datahubClient } }) => {
+  const [state, setState] = useState({ loading: true });
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const dailyVisitsData = await datahubClient.getDailyVisitsByPeriod({
+        domainName: domainName,
+        dateFrom: dateFrom,
+      });
+      if (!dailyVisitsData.success || !dailyVisitsData.value) {
+        setState({ loading: false });
+      } else {
+        const data = dailyVisitsData.value.map((item) => {
+          let date = new Date(item.from);
+          date = date.getFullYear() + '-' + (date.getMonth() + 1) + '-' + date.getDate();
```

Comment:
Please, use a kind of formatter in place of manually doing this, I think that you should also apply _intl_ here.

Reply:
if i move to datahub client can use `intl` in the service?

Reply:
No, you are using the same `date` variable for two different things (for that reason is a good idea trying to avoid `let` in favor of `const`).

Parsing the API response string field value as a date is a responsibility of the service. Formatting the date in the desired format is a responsibility of the UI component and it could take advantage of _Intl_ if need.

Reply:
fixed